includes/Core/Modules/REST_Modules_Controller.php

```diff
@@ -686,6 +688,27 @@ private function get_rest_routes() {
 									continue;
 								}
 
+								// Since currently the Analytics_4 module doesn't have an ownerID setting,
+								// it uses the ownerID from Analytics as the source of truth. Hence,
+								// instead of updating ownerID for Analytics_4, we should be updating that
+								// of Analytics.
+								if ( Analytics_4::MODULE_SLUG === $slug ) {
```

Comment:
Thanks for taking care of this and adding an explanation. However, since we have some test coverage for the `recover-modules` endpoint, would you be able to add some coverage for this behavior as well?

Reply:
Added, thank you!